helm/hpcc/values.yaml

```diff
@@ -511,7 +511,9 @@ sasha:
     #switchMinTime: 1
     #queues: "*"
 
-  xref: {} # NB: no properties defined, use defaults
+  xref:
+    eclwatchProvider: true
```

Comment:
I suspect it might be better to default this to true in the code in k8s, and remove from here.
I suspect it will soon be the only option.